src/Microsoft.Data.Analysis/TextFieldParser.cs

```diff
@@ -47,7 +144,13 @@ internal class TextFieldParser : IDisposable
 
         private string[] _delimitersCopy;
 
-        private Regex _whiteSpaceRegEx = new Regex("\\s", RegexOptions.CultureInvariant);
+        private Regex _delimiterRegex;
+
+        private Regex _delimiterWithEndCharsRegex;
+
+        private int[] _whitespaceCodes = new int[] { '\u0020' };
```

Comment:
there are a lot more than this in

[https://source.dot.net/#Microsoft.VisualBasic.Core/Microsoft/VisualBasic/FileIO/TextFieldParser.vb,1468](https://source.dot.net/#Microsoft.VisualBasic.Core/Microsoft/VisualBasic/FileIO/TextFieldParser.vb,1468)

Why did we only limit it to `\u0020`?

Reply:
I had initially reduced it to just `0020` to test the code. Added all of them back now. Some of them don't really look like whitespace chars (for ex: `\u1680` returns ` ` in my interactive window) though. In any case, I expect usage of these characters to be very rare.